packages/changeset-validations/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@ember-headless-form/changeset",
+  "version": "0.0.0",
+  "description": "Integrates ember-changeset with ember-headless-form",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "https://github.com/CrowdStrike/ember-headless-form.git",
+  "license": "MIT",
+  "author": "CrowdStrike UX Team",
+  "files": [
+    "addon-main.cjs",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
+    "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
+    "lint:js": "eslint . --cache",
+    "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
+    "lint:js:fix": "eslint . --fix",
+    "lint:types": "glint",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
+    "prepack": "rollup --config"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.0.0"
+  },
+  "peerDependencies": {
+    "validated-changeset": "^1.3.4",
+    "ember-changeset": "^4.1.2",
+    "ember-headless-form": "^0.0.0"
```

Comment:
This _doesn't_ need to use the `workspace:../` notation as it's a separate package built on top of `ember-headless-form` correct?  (Just trying to wrap my head around that concept a bit more!)

Reply:
I don't think you can use it here, because the `peerDependencies` is a field that users will use when installing the package, so it cannot refer to our monorepo. We have `workspace:*` for `devDependencies` though, which is only used internally, and by which we say that we don't want the package installed from npm under any circumstance, but to always use/link the code that's in the monorepo.